src/main/java/uk/gov/hmcts/reform/locationrefdata/configuration/DataQualityCheckConfiguration.java

```diff
@@ -9,5 +9,13 @@
 public class DataQualityCheckConfiguration {
 
     @Value("${zero-byte-characters}")
-    public List<String> zeroByteCharacters;
+    private List<String> zeroByteCharacters;
+
+    public List<String> getZeroByteCharacters() {
+        return zeroByteCharacters;
+    }
+
+    public void setZeroByteCharacters(List<String> zeroByteCharacters) {
+        this.zeroByteCharacters = zeroByteCharacters;
+    }
```

Comment:
a setter is not needed, let spring inject value into private field (setter method would also have to be unit tested if not removed). please consider removing setter method.